Services/MovieDataService.swift

```diff
@@ -38,7 +38,16 @@ class MovieDataService: MovieDataServiceProtocol {
     
     func getPlayingNowMoviesList(completion: @escaping (Result<[Movie], MovieServiceError>)-> Void) {
         guard isPlayingNowRequestCompleted else { return }
-        guard let url = URL(string: "https://api.themoviedb.org/3/movie/popular?api_key=b2b14caf40262a9c19a366b15e4e3537&language=en-US&page=\(playingNowPage)") else { return }
+        
+        var urlComponents = URLComponents()
+        urlComponents.scheme = "https"
+        urlComponents.host = "api.themoviedb.org"
+        urlComponents.path = "/3/movie/now_playing"
+        urlComponents.queryItems = [
+            URLQueryItem(name: "api_key", value: "b2b14caf40262a9c19a366b15e4e3537"),
```

Comment:
Can we move the API Key to be in a single place? Maybe a private static property, that way we are not risking a typo or in case we need to change the key it will be in a single place.

Reply:
Should I do the same with host?

I've also seen people add a separate enum or struct with constants. Is it a good idea?

Reply:
Yep, usually those are defined in a "configuration" place which can be a config file (json, plist) or a swift file with constants. You can create structs or enums to separate them as well, to define a "namespace" to organize it. When creating namespaces I prefer going with an enum instead of a struct, but both would work.